src/formatting.ts

```diff
@@ -1,7 +1,18 @@
 import FormData from "form-data";
+import { Readable } from "stream";
 
 export type SimpleObject = { [key: string]: unknown };
 
+type ContentsAndOptions = {
+  contents: Readable;
+  knownLength?: number;
```

Comment:
We're not exampling the usage of `knownLength` in the readme. I appreciate it's optional so we might not want to, but I thought I'd highlight it as we are for the other optional params.

https://github.com/onfido/onfido-node/pull/18/files#diff-04c6e90faac2675aa89e2176d2eec7d8R171-R175

Reply:
I wasn't sure if it was worth documenting `knownLength` because I can't think of any situations where it will need to be given, and thought adding it to the example would cause more confusion, but I wanted to make the option available since form-data supports it. I think I might remove it, along with `filename`.

Reply:
Make sense @ciaran16, let's go ahead and remove those then.